### ISSUE TITLE
How to unhide hidden content from between tags?

### ISSUE THREAD

**Issue by shanginn** [f0rb1dd3n/Reptile#1]
Hey, great work! thank you!
But how to unhide content inside <reptile> tags after it's been inserted?

**Comment by shanginn** [comment]
and it seems like `installer.sh` is failing after installing because of that

**Comment by f0rb1dd3n** [comment]
Hello, thank you for your words. 

Reptile hide contents by hooking syscall read(). So, to unhide, you can overwrite the lines of the tags, and the next time read() is called, the contents won't be hidden. Else, you must to remove the module.

Installer.sh loads the module at the end. So, nothing is hidden until that. What distro are you installing, I will verify this bug.

Greetings,
Ighor

**Comment by f0rb1dd3n** [comment]
@shanginn did you solve this issue? Can I close this?

**Comment by shanginn** [comment]
hello, sorry for the delay.
installer.sh failing after the install.
my question is: how can I overwrite lines inside the tags if I can't see them?

**Comment by shanginn** [comment]
btw: you can see these lines using grep :)

**Comment by f0rb1dd3n** [comment]
I havent getting this error with install.sh, but I would like to test that. After install rootkit will act and you may get some erros. I think you must unload the module from kernel to use install.sh again.

About the tags, you can mark the line that you are putting that. Or as you said, you can use grep for that.  

By the way, thanks for your pull request. I merged that, but now I took out hacking.h and put that functions for its respective files that will be used. 

:)

**Comment by f0rb1dd3n** [comment]
@shanginn can we close this issue?

**Comment by shanginn** [comment]
I didn't quite understood what you meant about "marking" the lines with tags.
but I can't think about any neat solution to this anyway so sure we can close it. thank you for your time :)

**Comment by gr4yg00s3** [comment]
"I haven't getting this error with install.sh, but I  would like to test that. After installing, Reptile will act and you may  get some errors. I think you must unload the module from kernel to use  install.sh again."- Hey, I've just started playing around with this, and I came across the same problem. Please can you help me out with how should I unload the module? Because it says cannot find file when I run "sudo rmmod /lib/reptile/reptile_mod.ko"

**Comment by f0rb1dd3n** [comment]
I mean, when you are writing the tags on a file, you take a note what line you are writing in. But using `grep -n "<reptile" file.txt` you will can see this line, if want to estract the exact lines that starts the tags you can do:

`grep -n "<reptile" file | cut -d ":" -f 1`

But now, you haven't to do it anymore. I just wrote a feature to enable and disable "hiding file content feature". All you have to do is:

`kill -51 0`

That command will enable "hiding file content feature" if is disabled, or disable if is enabled.

I hope you enjoy this. 
Please, tell me if I solved your issue.

Cya

**Comment by shanginn** [comment]
yes, it's great feature. but it could potentially disclosure presence of the rootkit.
I know I'm asking too much, but is it possible to temporarily disable hiding for specific process, something like `sudo` for this feature. for example `rdo vim file.txt` to be able to edit hidden content of the `file.txt` without unhiding content of all files?
